pytest/lib/cluster.py

```diff
@@ -777,9 +777,11 @@ def spin_up_node_and_push(i, boot_node: BootNode):
     return nodes
 
 
+ROOT_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.realpath(__file__))))
+
 DEFAULT_CONFIG = {
     'local': True,
-    'near_root': os.environ.get("NEAR_ROOT", '../target/debug/'),
+    'near_root': os.environ.get("NEAR_ROOT", os.path.join(ROOT_DIR, 'target/debug')),
```

Comment:
`pathlib` is more readable:

    REPO_DIR = pathlib.Path(__file__).resolve().parent.parent.parent
    NEAR_ROOT = os.environ.get('NEAR_ROOT) or (REPO_DIR / 'target/debug')

    'near_root': NEAR_ROOT